.ci.yaml

```diff
@@ -195,7 +195,7 @@ targets:
       dependencies: >-
         [
           {"dependency": "android_sdk"},
-          {"dependency": "chrome_and_driver"}
+          {"dependency": "chrome_and_driver", "version": "version:84"},
```

Comment:
The roller builder failed. We need to remove the comma in the end if it is the last element.

Reply:
fixed